### ISSUE TITLE
.cia version crashes on N3DS; works on O3DS.

### ISSUE THREAD

**Issue by AileSc** [mtheall/ftpd#48]
I compiled the latest commit as a .cia (both with and without #47 fixed), and it crashes to the "An error has occurred. Hold down the ..." screen for me and a few others. However, people in the #cakey IRC channel reported that the .CIA I compiled works fine on O3DS, so this issue appears to be N3DS only; might be USA only as well, no one with a EUR/JPN unit was around to test. ErrDisp seems to say that the Home Menu is complaining about fs not finding something: 
```
FATAL ERROR 

PID: 0x00040030_00008F02
REV: 61806
AID: 0xFFFFFFFF_FFFFFFFF
ADR: 0x00229338
RSL: 0xC8804478
          Level: -7 (this should be 25, some sort of display error)
          Summary: 4
          Module: 17
          Desc: 120
```

**Comment by mtheall** [comment]
I don't have a n3ds. I don't know how to troubleshoot this. If someone can fix it or at least give me more specific information with this error message, please let me know or make a PR.

**Comment by mtheall** [comment]
Fixed with d18ce24f
